packages/web/src/pages/sign-up-page/pages/CreateLoginDetails.tsx

```diff
@@ -79,66 +59,32 @@ export const CreateLoginDetailsPage = () => {
       validationSchema={loginDetailsFormikSchema}
     >
       {({ isValid, dirty }) => (
-        <Flex
-          as={Form}
-          direction='column'
-          pv={isMobile ? 'xl' : '2xl'}
-          ph={isMobile ? 'l' : '2xl'}
-          h='100%'
-          justifyContent='space-between'
-        >
-          <Flex direction='column' gap='2xl'>
-            <Flex direction='column' gap='s'>
-              <Text
-                color='heading'
-                size={isMobile ? 'm' : 'l'}
-                strength='default'
-                variant='heading'
-              >
-                {messages.title}
-              </Text>
-              <Text color='default' size={isMobile ? 'm' : 'l'} variant='body'>
-                {messages.description}
-              </Text>
-            </Flex>
-            <Flex direction='column' h='100%' gap='l'>
-              <Flex direction='column' gap='l'>
-                <HarmonyTextField
-                  name='email'
-                  autoComplete='email'
-                  label={messages.emailLabel}
-                />
-                <PasswordField name='password' label={messages.passwordLabel} />
-                <PasswordField
-                  name='confirmPassword'
-                  label={messages.confirmPasswordLabel}
-                />
-                <CompletionChecklist />
-              </Flex>
+        <Page as={Form}>
+          <Heading
+            heading={messages.title}
+            description={messages.description}
+          />
+          <Flex direction='column' h='100%' gap='l'>
+            <Flex direction='column' gap='l'>
+              <HarmonyTextField
+                name='email'
+                autoComplete='email'
+                label={messages.emailLabel}
+              />
+              <PasswordField name='password' label={messages.passwordLabel} />
+              <PasswordField
+                name='confirmPassword'
+                label={messages.confirmPasswordLabel}
+              />
+              <CompletionChecklist />
```

Comment:
so much nicer 👏

Reply:
thanks! yeah it made things 3x easier to read imo